metron-platform/metron-parsing/metron-parsing-storm/pom.xml

```diff
@@ -36,6 +37,28 @@
       <artifactId>commons-lang3</artifactId>
       <version>3.2</version>
     </dependency>
+    <dependency>
+      <groupId>org.apache.curator</groupId>
+      <artifactId>curator-client</artifactId>
+      <version>${global_curator_version}</version>
+      <exclusions>
+        <exclusion>
+          <groupId>org.apache.zookeeper</groupId>
+          <artifactId>zookeeper</artifactId>
+        </exclusion>
+      </exclusions>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.curator</groupId>
+      <artifactId>curator-recipes</artifactId>
+      <version>${global_curator_version}</version>
+      <exclusions>
+        <exclusion>
+          <groupId>org.apache.zookeeper</groupId>
```

Comment:
To match what we did in #1515, can you add this comment above `<groupId>`.
```
<!-- need to exclude zookeeper 3.5.x http://curator.apache.org/zk-compatibility.html -->
```

Reply:
If we decide we still need this, I will add this comment.